store/tikv/2pc.go

```diff
@@ -589,7 +592,12 @@ func (c *twoPhaseCommitter) execute() error {
 		return errors.Annotate(err, txnRetryableMark)
 	}
 
-	err = c.commitKeys(NewBackoffer(commitMaxBackoff, ctx), c.keys)
+	backoffTimeout := commitMaxBackoff
+	if testingCommit {
+		// Set to a small value to accelerate testing.
+		backoffTimeout = 2000
```

Comment:
it's not exposed to testing, the inteface is `Commit()`. we can just change the behavior via global variable.